clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2950,6 +2950,29 @@ max_concurrency, initiation_interval, or ivdep.
   }];
 }
 
+def SYCLIntelFPGALoopCountAvgAttrDocs : Documentation {
+  let Category = DocCatVariable;
+  let Heading = "intel::loo_control_avg";
+  let Content = [{
+This attribute applies to a loop. Its purpose is to pass to the optimizer the
+average trip count of a loop. This will allow the optimizer to figure out if
```

Comment:
Do I understand properly that this attribute exists because the optimizer isn't able to figure out the average trip count of a loop and so the hope is that the user will know the answer? Isn't this what profile-guided optimizations are for? (IOW, does this attribute really need to exist?)

Reply:
I have updated the doc for the attribute. That might be a little more clear now?